src/Raven.Server/Commercial/LicenseManager.cs

```diff
@@ -939,27 +940,35 @@ private void VerifyCoresPerNode(Dictionary<string, DetailsPerNode> detailsPerNod
                 {
                     nodeDetails.Value.UtilizedCores = coresPerNode;
                 }
+                
+                return;
             }
-            else
+
+            if (licenseChanged)
```

Comment:
Only if the number of cores in the license was changed (if `MaxCores` was changed)

Reply:
done